tests/bears/BearTest.py

```diff
@@ -142,22 +143,30 @@ class BearTest(BearTestBase):
 
     def test_default_aspects(self):
         assert type(Bear.aspects) is defaultdict
+        assert type(Bear.aspects['detect']) is aspectlist
+        assert type(Bear.aspects['fix']) is aspectlist
         assert Bear.aspects['detect'] == Bear.aspects['fix'] == []
 
     def test_no_fix_aspects(self):
         assert type(aspectsDetectOnlyTestBear.aspects) is defaultdict
+        assert type(aspectsDetectOnlyTestBear.aspects['detect']) is aspectlist
+        assert type(aspectsDetectOnlyTestBear.aspects['fix']) is aspectlist
         assert aspectsDetectOnlyTestBear.aspects['fix'] == []
         assert (aspectsDetectOnlyTestBear.aspects['detect'] ==
                 [CommitMessage.Shortlog.ColonExistence])
 
     def test_no_detect_aspects(self):
         assert type(aspectsFixOnlyTestBear.aspects) is defaultdict
+        assert type(aspectsFixOnlyTestBear.aspects['detect']) is aspectlist
+        assert type(aspectsFixOnlyTestBear.aspects['fix']) is aspectlist
         assert aspectsFixOnlyTestBear.aspects['detect'] == []
         assert (aspectsFixOnlyTestBear.aspects['fix'] ==
```

Comment:
I think you should add `assert CommitMessage.Shortlog.TrailingPeriod.some_subaspect in aspectsFixOnlyTestBear.aspects['fix']` here